packages/actions/test/unit/storage.test.ts

```diff
@@ -132,43 +128,42 @@ describe("Storage", () => {
                     bucketName,
                     objectName,
                     localPath,
-                    streamChunkSizeInMb.toString(),
-                    presignedUrlExpirationInSeconds
+                    streamChunkSizeInMb
                 )
                 expect(success).to.be.true
```

Comment:
let's remove the check to be true, and instead check that the promise is fulfilled without throwing